### PR TITLE
At A Glance

### PR DESCRIPTION
**Pull request by minhkhang1795**
I implemented based on our design but tweaked some stuff (colors, positions, etc.). Totally not accessible for screen readers for now!

### CODE REVIEW

**Review by ntchirkova — COMMENTED** [review]
Code looks to me, next team should look into colors... I am not sure how I feel about the black bar for when we don't know thresholds, cause it can come off as aggressive. Maybe keep it as the same green when below the threshold.

**Comment by anushadatar** [comment]
+1 on liking the idea of using green when we don't know the thresholds, otherwise LGTM

File: src/Utils.js [hunk 1]
```diff
@@ -76,49 +76,61 @@ export const Pollutants = {
         id: "pm25",
         name: "PM25",
         unit: "\xB5g/m\u00B3",
-        safe: "??",
+        baseline: 35,
```
Comment (anushadatar) [inline src/Utils.js:79]: Do we have a source for all of these values? might just be good to have handy moving forward

**Review by anushadatar — COMMENTED** [review]
Generally looks good to me, would be interested in adding a source and also would agree with Nina's idea about keeping it green if we don't know the threshold. This is really awesome work though!

**Comment by Inkering** [comment]
Code looks good, but did we get confirmation from Aria that the learn-more's are good to go? // confirmation that we have the latest version of those edits she's been working on in this PR?

**Comment by ntchirkova** [comment]
Checked-in with Aria, she said it is ready. Said she handed it off to Khang so we should have the latest version.

**Comment by Inkering** [comment]
rad!

File: src/Utils.js (hunk 1 above)
Comment (minhkhang1795) [reply src/Utils.js:79]: Pollutants thresholds: https://docs.google.com/document/d/1DAdqBDDwUHGc4v8tjZyKpxbMy2CmlfGCQtZPeJ7fZ4M/edit

**Pull request merged**